thruster/examples/hyper_most_basic.rs

```diff
@@ -14,12 +14,13 @@ async fn plaintext(mut context: Ctx, _next: MiddlewareNext<Ctx>) -> Ctx {
     context
 }
 
-fn main() {
+#[tokio::main]
```

Comment:
Let's keep this and the hello world with the `start` syntax, as these should be as simple + easy as possible.